### ISSUE TITLE
Decoder plugin not working and giving errors

### ISSUE THREAD

**Issue by chrismeteu** [atteneder/DracoUnityDemo#4]
I'll start by saying that my app will need to be built on iOS devices. I have opened this repo on both a PC and a Mac; here are my findings:

PC - Unity 2021.3.9f1, burst 1.6.6, and Draco 4.0.2
After I hit play in the Editor I get 5 duplicate error messages: "A Native Collection has not been disposed, resulting in a memory leak. Enable Full StackTraces to get more details.". Unfortunately no new info is provided after I turned on Full Stack Tracing. Also, the bunny is never loaded with either the DracoDemo or the DracoDemoMeshData components, No other errors so I don't have much info to provide.

Mac - Unity 2019.4.29f1, burst 1.6.6, and Draco 4.0.2 and Unity 2021.3.9f1, burst 1.4.11, and Draco 3.2.0
I went through the steps of canceling, enabling in prefs, and then reopening Unity (I hit cancel about 5 times the first time, but only need to hit ok once after enabling). When I run in the editor I get a ton of errors, the first of which is still the missing bundle file so I don't think the previous steps fully worked.
Another thing I tried was using cmake and building the plugin myself, but it didn't seem to make a difference. (I'll admit I'm not experienced with cmake files and saw conflicting examples on yours and the Google Draco repo)

Please let me know what other info might be helpful or if I need to explain anything in more detail.

**Comment by chrismeteu** [comment]
Here's an update. I reinstalled Unity versions 2019, 2020, and 2021 on my PC. I was finally able to run successfully after opening in 2019.4.40. Upgrading to Draco 3.3.2 also continued to work. I then upgraded to 2020.3.18 and it worked. I upgraded again to 2021.9 and Draco to 4.0.2 without issues. So starting at the beginning and manually upgrading 1 by 1 seemed to fix any issues when running on my PC. Opening the repo directly into 2021 still had issues so something about upgrading too fast seems to not work in Unity (at least for me). I'll try this on Mac next.
